keyboards/converter/att56k_usb/keymaps/default/keymap.c

```diff
@@ -0,0 +1,34 @@
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "keycode.h"
+#include "print.h"
+#include "debug.h"
+#include "util.h"
+#include "att56k.h"
```

Comment:
This should be replaced with: 
```suggestion
#include QMK_KEYBOARD_H
```